### DISCUSSION TITLE
Where is the backup tab?

### DISCUSSION THREAD

**Discussion by igmarketing** [Dokploy/dokploy#535] [Q&A]
Not sure if I am dumb but... I can't find the backup tab 😅 According to the [documentation](https://docs.dokploy.com/en/docs/core/databases/backups), the tab "Backups" should be in the Settings but I don't see it anywhere. It's a fresh install on Ubuntu 24.04.

![dokploy-settings](https://github.com/user-attachments/assets/4031b709-9977-4727-a6b3-97d4b7bf0cbf)

**Comment by Siumauricio** [comment]
Oh we made a mistake here, it was S3 Destinations in the settings section.

**Reply by igmarketing** [reply]
Got it but where are the backups settings? We just have the option to add a storage provider but we don't see any option for backups or even restoring?

**Reply by Siumauricio** [reply]
Backups are only for databases, restoring is not possible yet
